.github/workflows/build-interop-docker.yml

```
@@ -19,9 +25,14 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: set tag name
+        if: github.event_name == 'tag'
+        # Tagged releases won't be picked up by the interop runner automatically,
+        # but they can be useful when debugging regressions.
+        run: echo "TAG=${{ github.event.release.tag_name }}"
       - uses: docker/build-push-action@v4
         with:
           context: "{{defaultContext}}:interop"
           platforms: linux/amd64,linux/arm64
           push: true
-          tags: martenseemann/quic-go-interop:latest
+          tags: martenseemann/quic-go-interop:${{ env.TAG }}
```

With my proposed changes, we should update where we get the tag from too.

```suggestion
          tags: martenseemann/quic-go-interop:${{ steps.tag.outputs.tag }}
```